Assets/Scripts/Level generation scripts/TerrainGeneratorScript.cs

```diff
@@ -7,12 +7,15 @@
 public class TerrainGeneratorScript : MonoBehaviour
 {
     private readonly float _tileWidth = 10.0f;
-    
+    private const string SpawnPointName = "NPC Spawn point";
+
     [SerializeField] private int blocks = 2; //number of blocks being made, block x block
     [SerializeField] private int size = 5;
     [SerializeField] private float tavernPercent = 0.25f;
     [SerializeField] private GameObject outerTile;
-    
+    [SerializeField] private List<GameObject> _NPCs;
+    private static List<GameObject> NPCs;
```

Comment:
Why do we have two variables that are essentially the same? Is it because we need it to be static?

Reply:
Yeah
I need it to be static to access it in the function, because the function is static 
And the function is static because I will call it from another script when you pick a beer

I could have done it differently, but i found this easier to do